Environment/Environment.pillar

```diff
@@ -442,43 +443,42 @@ your image first!
 !!!The Inspector
 
 (do this) As an illustration of what you can do with an inspector, type
-==TimeStamp now== in a workspace, and then right-click and choose ==Inspect
-it==.
+==DateAndTime now== in a playground, and then right-click and choose ==Inspect
+it== or ==Do it and go==.
```

Comment:
What does 'Do it and go' actually do? Since we haven't mentioned that action earlier in the book (I actually didn't know it existed until I just read this :) ), we should probably explain it, either here or earlier in the book.